pkg/api/cluster_types.go

```diff
@@ -193,6 +197,12 @@ type StrimziVersion struct {
 	KafkaIBPVersions []KafkaIBPVersion `json:"kafkaIBPVersions"`
 }
 
+type DynamicCapacityInfo struct {
+	MaxNodes       int32 `json:"max_nodes"`
```

Comment:
Can we be more specific with the naming? are we talking about compute nodes? worker nodes? infra nodes?

Reply:
It is the maximum worker nodes.